topi/python/topi/cuda/conv2d_winograd.py

```diff
@@ -375,6 +375,12 @@ def _alter_conv2d_layout(attrs, inputs, tinfos):
         if cfg.template_key == 'direct':
             return None
 
+        if cfg.template_key == 'int8':
+            new_attrs['layout'] = 'NCHW4c'
+            new_attrs['out_layout'] = 'NCHW4c'
+            new_attrs['kernel_layout'] = 'OIHW4o4i'
+            return sym.contrib.conv2d_NCHWc_int8_prepacked(*copy_inputs, **new_attrs)
```

Comment:
Can we just return sym.conv2d?
I think this new symbol is redundant.

We need new symbol for winograd because we have to use a different infer_shape. But for conv2d, the infer_shape in nnvm already supports these layouts.

Reply:
conv2d currently only accept NCHW / NHWC. If we pass NCHW4c, currently will meet layout assert error.

Reply:
Do you mean this ?
 https://github.com/dmlc/tvm/blob/ec0d497c69ca307fb998c3d81c0a7e48bb5f18d6/nnvm/python/nnvm/top/nn.py#L140-L146

It's better to change this instead of defining new op

Reply:
@vinx13  Yes, the problem is the one @FrozenGene just pointed out. I think we can fix it.

Reply:
@merrymercy another problem is that we can't call a topi template directly with packed layout, instead I registered a workload function to create original workload (in NCHW layout) from packed input. 
Adding this works.
https://github.com/dmlc/tvm/blob/ecad8bf05e80804218f8ef02bbc5c4337d247783/nnvm/python/nnvm/top/nn.py#L108